### PR TITLE
[WFLY-16723] Correct list of artifacts distributed on README-EJB-JMS.txt

### PR DESCRIPTION
**Pull request by sudeshnas93**
Issue: https://issues.redhat.com/browse/WFLY-16723

### CODE REVIEW

**Comment by chengfang** [comment]
I think it will be good to preserve the previous ordering: standard API artifacts first, followed by one blank line, then the rest artifacts roughly alphabetically.

**Comment by sudeshnas93** [comment]
Made the changes. Please review. @chengfang

File: ee-feature-pack/common/src/main/resources/content/bin/client/README-EJB-JMS.txt [hunk 1]
```diff
@@ -8,26 +8,69 @@ org.jboss.spec.javax.jms:jboss-jms-api_2.0_spec
 org.jboss.spec.javax.transaction:jboss-transaction-api_1.3_spec
```
Comment (chengfang) [inline ee-feature-pack/common/src/main/resources/content/bin/client/README-EJB-JMS.txt:8]: the above 3 lines should be removed. 

the 4 jakarta.* api artifacts should be placed here instead. See next comment.

File: ee-feature-pack/common/src/main/resources/content/bin/client/README-EJB-JMS.txt [hunk 2]
```diff
@@ -8,26 +8,69 @@ org.jboss.spec.javax.jms:jboss-jms-api_2.0_spec
...
+io.netty:netty-transport-classes-epoll
+io.netty:netty-transport-native-epoll:linux-x86_64
+io.netty:netty-transport-native-epoll:linux-aarch_64
+io.netty:netty-transport-classes-kqueue
+io.netty:netty-codec-http
+io.undertow:undertow-core
+jakarta.jms:jakarta.jms-api
+jakarta.transaction:jakarta.transaction-api
+jakarta.ejb:jakarta.ejb-api
+jakarta.resource:jakarta.resource-api
```
Comment (chengfang) [inline ee-feature-pack/common/src/main/resources/content/bin/client/README-EJB-JMS.txt:34]: the above 4 jakarta api should move to the top to replace the no-longer-used javax api artifacts.

**Comment by bstansberry** [comment]
@sudeshnas93 will you be able to make the requested changes on Monday, Oct 31? I want to wrap up merging things for WF 27 tomorrow.

**Comment by sudeshnas93** [comment]
> @sudeshnas93 will you be able to make the requested changes on Monday, Oct 31? I want to wrap up merging things for WF 27 tomorrow.

It will be done by today.

File: ee-feature-pack/common/src/main/resources/content/bin/client/README-EJB-JMS.txt (hunk 2 above)
Comment (sudeshnas93) [reply ee-feature-pack/common/src/main/resources/content/bin/client/README-EJB-JMS.txt]: Made the required changes.

**Comment by bstansberry** [comment]
Thanks @sudeshnas93

**Pull request merged**